cloud/src/autogluon/cloud/scripts/train.py

```diff
@@ -132,3 +133,9 @@ def get_env_if_present(name):
         if config.get("leaderboard", False):
             lb = predictor.leaderboard(silent=False)
             lb.to_csv(f"{args.output_data_dir}/leaderboard.csv")
+
+    print('Saving serving script')
+    serving_script_saving_path = os.path.join(save_path, 'code')
+    os.mkdir(serving_script_saving_path)
+    serving_script_path = get_input_path(args.serving_script)
+    shutil.move(serving_script_path, os.path.join(serving_script_saving_path, os.path.basename(serving_script_path)))
```

Comment:
do we want to move or copy?

Reply:
Move is fine. We don't use this script while training. Just save it as part of the model artifacts so that we don't need to repack the inference code later.